index.js

```diff
@@ -35,10 +35,14 @@ module.exports = function(html, options) {
 
   /* head */
 
+  /* main amp library */
+  $('head script[src="https://cdn.ampproject.org/v0.js"]').remove();
+  $('head').prepend('<script async src="https://cdn.ampproject.org/v0.js"></script>');
```

Comment:
I use `prepend` so that `meta` will be the first element of `head`, and `script` will be the second